src/main/java/com/google/sps/model/Events.java

```diff
@@ -0,0 +1,22 @@
+package com.google.sps.model;
+
+import java.util.List;
+import java.lang.Long;
+
+public class Events {
```

Comment:
Since this represents a single Event, call it Event instead of Events?

Add an id too. This will be a unique random number that you can store in Datastore, and will be passed around as id.